lib/shopify_app/configuration.rb

```diff
@@ -81,6 +83,14 @@ def has_scripttags?
     def enable_same_site_none
       !Rails.env.test? && (@enable_same_site_none.nil? ? embedded_app? : @enable_same_site_none)
     end
+
+    def shop_access_scopes
+      @shop_access_scopes || scope
```

Comment:
It seems a little odd to return a singular item `scope` when the method name implies a collection.

Reply:
Agreed! However, the `scope` attribute has been used to define a collection of scopes stored in a string for a while now. Changing this will be a breaking change for shopify_app developers.

Reply:
Good point, we can mark this as future work to update `scope` to `scopes`.